src/H5CX.c

```diff
@@ -2900,6 +2900,40 @@ H5CX_set_mpio_actual_chunk_opt(H5D_mpio_actual_chunk_opt_mode_t mpio_actual_chun
     FUNC_LEAVE_NOAPI_VOID
 } /* end H5CX_set_mpio_actual_chunk_opt() */
 
+/*-------------------------------------------------------------------------
```

Comment:
Added these since this should be more convenient than get/or/set. I think we could possibly use these in a few more places instead of tracking the property in a separate variable.